dotcom-rendering/src/web/components/FixedSmallSlowVMPU.tsx

```diff
@@ -26,7 +26,7 @@ export const FixedSmallSlowVMPU = ({
 		<UL direction="row">
 			{firstSlice33.map((trail) => (
 				<LI percentage="33.333%" padSides={true} key={trail.url}>
-					<FrontCard
+					<Card25Media25Tall
```

Comment:
As the % this card is taking up is 33%, perhaps we should create a duplicate of `Card25Media25Tall` in `cardWrappers.tsx` and rename it to `Card33Media33Tall` so that the naming scheme matches up?